tests/integration/test_sharding.py

```diff
@@ -1119,9 +1118,9 @@ def validate_shards(cluster_shards):
             else:
                 assert False, "didn't match %s" % cluster_shards
 
-
     validate_shards(cluster.node(1).execute('CLUSTER', 'SHARDS'))
 
+
```

Comment:
Changes in this file are leftover right, not related

Reply:
not related, but I think bug fixes.  though this extra line is removable.

Reply:
actually, no this extra line was also needed for py linting